client/Packages/com.beamable/Samples/LightBeamSamples/LootBox/BeamableServices~/LootBoxService/LootBoxService.cs

```diff
@@ -0,0 +1,46 @@
+using Beamable.Common;
+using Beamable.Server;
+using System;
+using System.Globalization;
+
+namespace Beamable.LootBoxService
+{
+	[Microservice("LootBoxService")]

```

Comment:
Create a sibling folder of `BeamableServices~` with an assembly definition inside called `<beamoId>.Client` and make sure there is a empty script in there